civiclens/collect/access_api_data.py

```diff
@@ -68,13 +67,12 @@ def pull_reg_gov_data(
     end_date=None,
     params=None,
     print_remaining_requests=False,
-    wait_for_rate_reset=True,
```

Comment:
All these changes look great. Thanks for doing some cleanup! What is the reason for removing the rate reset? This is the only thing that seems like it could affect the pipeline.

Reply:
Yeah at the very least, as written, wait_for_rate_reset is used later in the function. Greg and I have and will be using that functionality to pull data moving forward

Reply:
for some reason, vsocde was telling me that arg was not being called later in the function but that obviously isn't true, will add back in!

Reply:
Greg recently made some updates to this code, so things may have gotten switched around in the merge

Reply:
Where do we stand on these changes @andrewjtdunn @GregoryHo88 ?

Reply:
We want to keep `wait_for_rate_reset` in the code